nixos/modules/services/x11/display-managers/gdm.nix

```diff
@@ -125,7 +125,6 @@ in
       "getty@tty1.service"
     ];
 
-    systemd.services.display-manager.conflicts = [ "getty@tty1.service" ];
```

Comment:
@lheckemann @teh what was the motivation for this in the first place?

Reply:
@teh answered that here: https://github.com/NixOS/nixpkgs/issues/21439#issuecomment-340207452 (it comes from upstream service file)

Reply:
But what if @GDM_INITIAL_VT@ is != 1. Would that fix it too?

Reply:
After re-reading the systemd behaviour of `Conflicts` IMO what should happen is that tty1 is started, and then replaced by GDM, and if GDM dies (the systemd unit fails) then tty1 should restart. I have a feeling that we're seeing some other interactions here but if removing that line works for people then I'm all for it! (better than a completely broken system, even if we don't necessarily understand why)